packages/ods/src/components/quantity/tests/navigation/ods-quantity.e2e.ts

```diff
@@ -28,6 +28,7 @@ describe('ods-quantity navigation', () => {
       await setup('<ods-quantity></ods-quantity>');
       const odsFocusSpy = await page.spyOnEvent('odsFocus');
       await page.keyboard.press('Tab');
+      await page.keyboard.press('Tab');
```

Comment:
why 2 tab?

Reply:
The button don't emit the focus event anymore